.github/workflows/build.yaml

```diff
@@ -26,34 +28,22 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  docs-build:
```

Comment:
you won't actually be building docs here, right?

Even if you do plan to, right now there isn't actually a `ci/build_docs.sh` checked into the repo, so this would immediately fail. Better to delete it and re-introduce it when you're ready, I think.

Reply:
Yes, there are no plans to build docs in this repo for this release